QModManager/Patching/QMod.cs

```diff
@@ -19,40 +19,40 @@ public QMod()
             // Empty public constructor for JSON
         }
 
-        [JsonProperty(Required = Required.Always)]
+        [JsonProperty(nameof(Id))]
```

Comment:
Is `JsonProperty` in the new version really don't use field/property name as its name by default?

Reply:
There is no empty constructor for in this version of `[JsonProperty]`, otherwise I would have used it.

Reply:
Hmm, what version of Json.NET BZ is using? There is an empty constructor for [JsonPropertyAttribute](https://github.com/JamesNK/Newtonsoft.Json/blob/666d9760719e5ec5b2a50046f7dbd6a1267c01c6/Src/Newtonsoft.Json/JsonPropertyAttribute.cs#L210) in sources.

Reply:
This conversation isn't going anywhere since you aren't looking at the DLL that came with Below Zero.